spec/Section 6 -- Execution.md

```diff
@@ -22,7 +22,7 @@ to be formatted according to the Response section below.
 To execute a request, the executor must have a parsed `Document` (as defined
 in the “Query Language” part of this spec) and a selected operation name to
 run if the document defines multiple operations, otherwise the document is
-expected to only contain a single operation. The result of the request is
+expected to only contain a single operation. How this information is passed to the executor is determined by the implementation. The result of the request is
```

Comment:
I think this addition is redundant and potential confusing. Of course this is the case, by definition an implementation's role is to implement the below algorithm, which describes the arguments which are expected to be provided